nuxt.config.js

```diff
@@ -197,6 +201,13 @@ export default {
         file: 'pt-BR.js',
         name: 'Português',
         domain: 'https://pt.nuxtjs.org'
+      },
```

Comment:
this lang cannot be put live yet until the subdomain has been created so for now you can leave it commented out.

Reply:
Sure